### ISSUE TITLE
Caption in Table (RAC)

### ISSUE THREAD

**Issue by aulonm** [adobe/react-spectrum#9015]
### Provide a general summary of the feature here

Would love the possibility to add Caption in the table component from react-aria-components

### 🤔 Expected Behavior?

React-aria-components should have an componet called TableCaption which adds a `<caption>` to the table, or something equivalent

### 😯 Current Behavior

At the moment it isn't possible to add `<caption>`-html tag inside the `<Table>`

### 💁 Possible Solution

A new component called `<TableCaption>` that would be possible to add within the `<Table>` component, the same way as we use TableBody/TableRow etc. 

### 🔦 Context

Because of accessability it would be nice to have the possibility to allow our users to use the caption tag if they want that or not

**Comment by LFDanLu** [comment]
@aulonm could you accomplish this with a external span + aria-labelledby/describedby? A virtualized table uses divs anyways so we'd have use something other than the caption element instead anyways.

**Comment by aulonm** [comment]
> [@aulonm](https://github.com/aulonm) could you accomplish this with a external span + aria-labelledby/describedby? A virtualized table uses divs anyways so we'd have use something other than the caption element instead anyways.

Sorry for the late reply! That is something we can absolutely do and a nice workaround. Thank you!
